.eslintrc.js

```diff
@@ -0,0 +1,199 @@
+module.exports = {
+  root: true,
+  overrides: [
+    {
+      /*
+       * REACT
+       */
+      files: ['**/*.js'],
+      env: {
+        es2020: true,
+        commonjs: true, // Needed to avoid import is reserved error
+        node: true,
+        browser: true,
+        jest: true,
+      },
+      parser: 'babel-eslint',
+      extends: [
+        'eslint:recommended',
+        'plugin:react/recommended',
+        'plugin:prettier/recommended',
+      ],
+      plugins: ['react', 'import'],
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true,
+        },
+        ecmaVersion: 2020,
+        sourceType: 'module',
+      },
+      rules: {
+        'react/prop-types': 0,
+        'prettier/prettier': [
+          'error',
+          {
+            singleQuote: true,
+            arrowParens: 'always',
+            semi: false,
+            bracketSpacing: true,
+            trailingComma: 'es5',
+            printWidth: 80,
+          },
+        ],
+      },
+      settings: {
+        react: {
+          version: 'detect',
+        },
+      },
+    },
+    {
+      /*
+       * JS
+       */
+      files: ['**/*.js'],
+      env: {
+        browser: true,
+        es2020: true,
+        'jest/globals': true,
+        node: true,
+      },
+      extends: [
+        'standard',
+        'plugin:prettier/recommended',
+        'eslint:recommended',
+        'plugin:cypress/recommended',
+      ],
+      plugins: ['jest'],
+      parserOptions: {
+        ecmaVersion: 2020,
+      },
+      rules: {
+        'cypress/no-unnecessary-waiting': 'off',
```

Comment:
Same as above, maybe it's not the role of the boilerplate to provide tests rules and plugins